controllers/category-controller.js

```diff
@@ -0,0 +1,9 @@
+const { fetchCategories } = require("../models/category-model");
+
+function getCategories(req, res) {
+  fetchCategories().then((categories) => {
+    res.status(200).send({ categories });
+  });
+}
```

Comment:
You should include a catch block here, otherwise you might find that your error handling doesn't work!